### PR TITLE
-add

### PR DESCRIPTION
**Pull request by Zapa-Mike**
footer, string resources
-change
button colours to black to match footer

### CODE REVIEW

File: src/app/components/footer/footer.component.css [hunk 1]
```diff
@@ -1,21 +1,177 @@
...
-    line-height: 1.1;
-    color: inherit;
-  }
\ No newline at end of file
+ 
+  .footer-distributed .footer-center i{
+    margin-left: 0;
+  }
+}
+.cc{
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.css:174]: please use an appropriate name that is more expressive than cc

File: src/app/components/footer/footer.component.css [hunk 2]
```diff
@@ -1,21 +1,177 @@
...
+  border-radius: 2px;
+ 
+  font-size: 20px;
+  color: #ffffff;
+  text-align: center;
+  line-height: 35px;
+ 
+  margin-right: 3px;
+  margin-bottom: 5px;
+}
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.css:151]: is this class even in use? try without it

File: src/app/components/footer/footer.component.css [hunk 3]
```diff
@@ -1,21 +1,177 @@
...
+ 
+/* If you don't want the footer to be responsive, remove these media queries */
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.css:153]: remove all unnecessary comments please

File: src/app/components/footer/footer.component.css [hunk 4]
```diff
@@ -1,21 +1,177 @@
...
+  }
+ 
+  .footer-distributed .footer-left,
+  .footer-distributed .footer-center,
+  .footer-distributed .footer-right{
+    display: block;
+    width: 100%;
+    margin-bottom: 40px;
+    text-align: center;
   }
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.css:168]: do we really need responsiveness?

File: src/app/components/footer/footer.component.html [hunk 5]
```diff
@@ -1,48 +1,41 @@
...
-        <ul class="dropdown-menu">
-          <li><a class="dropdown-item"> L'immeuble SGBC</a></li>
-          <li><a class="dropdown-item">Avenue des Banques</a></li>
-          <li>
-            <a class="dropdown-item"> {{ "app.footer.floor" | translate }}</a>
-          </li>
-        </ul>
-      </div>
+    <div>
+      <i class="fa fa-phone"></i>
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.html:19]: the i here doesn't do anything right?

Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.html:19]: the i here doesn't do anything right?

File: src/app/components/footer/footer.component.html [hunk 6]
```diff
@@ -1,48 +1,41 @@
...
-        <button type="button" class="btn btn-primary dropdown-toggle" #manage>
-          Email
-          <span class="caret"></span>
-        </button>
-        <ul class="dropdown-menu">
-          <li><a class="dropdown-item">info@britishteachingcentre.org</a></li>
-        </ul>
-      </div>
+    <div>
+      <i class="fa fa-phone"></i>
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.html:23]: the i here doesn't do anything right?

File: src/app/components/footer/footer.component.html [hunk 7]
```diff
@@ -1,48 +1,41 @@
...
+      <p>+237 242 66 53 78</p>
     </div>
-    <div class="col-md-3 col-lg-3">
-      <button class="btn btn-primary" routerLink="feedback">
-        {{ "app.footer.feedback" | translate }}
-      </button>
+    <div>
+      <i class="fa fa-envelope"></i>
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.html:27]: the i here doesn't do anything right?

File: src/app/components/footer/footer.component.css [hunk 8]
```diff
@@ -1,21 +1,177 @@
...
+  font-size: 14px;
+  font-weight: bold;
+  margin-bottom: 20px;
+}
+ 
+.footer-distributed .footer-icons{
+  margin-top: 25px;
+}
+ 
+.footer-distributed .footer-icons a{
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.css:136]: the css class footer-icons isn't used in the html at all right?

File: src/app/components/footer/footer.component.css (hunk 2 above)
Comment (Zapa-Mike) [reply src/app/components/footer/footer.component.css:151]: you're right it did nothing

File: src/app/components/footer/footer.component.html (hunk 7 above)
Comment (Zapa-Mike) [reply src/app/components/footer/footer.component.html:27]: Without it the text won't display

File: src/app/components/footer/footer.component.css [hunk 9]
```diff
@@ -1,21 +1,135 @@
...
+  width: 38px;
+  height: 38px;
+  border-radius: 50%;
+  text-align: center;
+  line-height: 42px;
+  margin: 10px 15px;
+  vertical-align: middle;
+}
+ 
+.footer-distributed .footer-center i.fa-envelope{
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.css:85]: fa-envelope not in use

File: src/app/components/footer/footer.component.css [hunk 10]
```diff
@@ -1,21 +1,135 @@
...
+ 
+.footer-distributed .footer-company-about span{
+  display: block;
+  color:  #ffffff;
+  font-size: 14px;
+  font-weight: bold;
+  margin-bottom: 20px;
+}
+  
+.facebookicon{
```
Comment (Youssef-Ayman) [inline src/app/components/footer/footer.component.css:132]: always dash between words

**Pull request merged**